src/Bus/Handler/PaymentStatusReceivedHandler.php

```diff
@@ -20,10 +21,9 @@
 use Sylius\Component\Core\Model\PaymentInterface;
 use Sylius\Component\Core\OrderCheckoutTransitions;
 use Sylius\Component\Resource\Repository\RepositoryInterface;
-use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
 use Symfony\Component\Messenger\MessageBusInterface;
 
-final class PaymentStatusReceivedHandler implements MessageHandlerInterface
+final class PaymentStatusReceivedHandler
```

Comment:
Why you removed the interface?

Reply:
The `MessageHandlerInterface` has been deprecated, likely since Symfony 5. To update the classes where this implementation has been removed, I can add the attribute `#[AsMessageHandler]`.

Reply:
Yup, I asked because I didn't see the attribute. And please be sure whether the attribute configuration is turned on the plugin.